src/app/modules/item/pages/item-log-view/item-log-view.component.ts

```diff
@@ -81,7 +83,7 @@ export class ItemLogViewComponent implements OnChanges, OnDestroy {
       {
         field: 'item.user',
         header: $localize`User`,
-        enabled: this.isWatchingGroup && [ 'Chapter', 'Task', 'Course' ].includes(type),
+        enabled: this.sessionService.isCurrentlyWatching && [ 'Chapter', 'Task', 'Course' ].includes(type),
```

Comment:
If you use observable to compute your states, you have to use them everywhere, but that should not be too difficult to pass this function a boolean.

Reply:
Fixed